templates/etcd-ha/3/README.md

```diff
@@ -1,41 +1,38 @@
 # Etcd 
 
-A distributed, highly-available key/value store written in Go.
+A distributed, highly-available key/value store for storing configuration information.
 
-### Info
+### General
 
- This creates an N-node etcd cluster on top of Rancher. The bootstrap process is performed using a standalone etcd discovery node. Upon the cluster entering a running state, this discovery service will shutdown. The state of the etcd cluster is a key/value pair stored on etcd itself, in a hidden key at location `/_state`.
+This template attempts to create an N-node Etcd cluster. Only 1 node is allowed per host. If not enough hosts are available, the largest cluster possible will be built with the resources available. Adding more hosts at a later date will result in the cluster scaling up to the maximum desired size.
```

Comment:
'attempts' makes it sound a bit unreliable. I get what the goal is, but maybe say something like: The template deploys a cluster with scheduling policy up to N nodes in 1 node increments ... 
Something to that end.